### PR TITLE
Feature/map 194/Change marker color for verified, unverified, reported, waiting to verification location

### PR DESCRIPTION
**Pull request by nikitius20**

### CODE REVIEW

File: frontend/src/redux/reducers/popupLocationReducer.tsx [hunk 1]
```diff
@@ -21,7 +21,9 @@ export const initialState: popupLocationStateType = {
     arrayPhotos: [],
     description: '',
     createdAt: new Date(),
-    updatedAt: new Date()
+    updatedAt: new Date(),
+
+    verificationStatus: ''
   }
 };
```
Comment (Yojuks) [inline frontend/src/redux/reducers/popupLocationReducer.tsx:29]: why you added 4 svg - maybe you can add 1 svg and can add className for cases?

**Review by Yojuks — APPROVED** [review]

File: frontend/src/components/LocationPopOut/LocationPopOut.tsx [hunk 2]
```diff
@@ -9,32 +9,52 @@ import {
   StyledMediaBox,
   StyledTooltip
 } from '../design/StyledLocationPopOut';
-import icon from '../../static/pin-308802.svg';
+import verifiedIcon from '../../static/verified-icon.svg';
+import unverifiedIcon from '../../static/unverified-icon.svg';
+import waitingIcon from '../../static/waiting-icon.svg';
 import img from '../../static/image-not-found.jpg';
+import { verificationStatusTypes } from '../../static/verificationStatusTypes';
```
Comment (obozhko-dev) [inline frontend/src/components/LocationPopOut/LocationPopOut.tsx:16]: For types we have separate file types.ts at the root of the folder front-end. I think this data related to types data and could be in types.ts

File: frontend/src/components/Map/DrawMarkerWhenLocationCreate/index.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Marker } from 'react-leaflet';
 import L from 'leaflet';
 import { latlngType } from '../../../../types';
-import icon from '../../../static/marker.png';
+import icon from '../../../static/creation-icon.svg';
 
 // import shadow from '../../../static/leaf-shadow.png';
```
Comment (obozhko-dev) [inline frontend/src/components/Map/DrawMarkerWhenLocationCreate/index.tsx:7]: remove, please, commented code if it didn't using

**Review by obozhko-dev — CHANGES_REQUESTED** [review]

File: frontend/src/components/Map/DrawMarkerWhenLocationCreate/index.tsx (hunk 3 above)
Comment (nikitius20) [reply frontend/src/components/Map/DrawMarkerWhenLocationCreate/index.tsx:7]: done

File: frontend/src/redux/reducers/popupLocationReducer.tsx (hunk 1 above)
Comment (nikitius20) [reply frontend/src/redux/reducers/popupLocationReducer.tsx:29]: you can't directly change color in svg.
You can filter a specific folor, but its too complicaed and inefficient, 
But if you want you can check it here https://codepen.io/sosuke/pen/Pjoqqp

**Review by obozhko-dev — APPROVED** [review]

**Pull request merged**